Src/BootCamp.Chapter/Lesson3.cs

```diff
@@ -0,0 +1,209 @@
+            float height = float.Parse(CheckStringForDigitsOnly());
+
+         
+            Console.WriteLine("Thank you for your information. You entered:");
+
+            Console.WriteLine($"\n``` {firstName} {lastName} is {age} years old, your weight is {weightInKg} kg and" +
+                              $" your height is {height} cm. ```");
+
+            
+            CalculateBmi(weightInKg, height);
+        }
+
+
+        //Make sure string has only letters
+        public static string CheckStringForLettersOnly()
+        {
+            bool stop = true;
+            string testString = "";
+
+            while (stop)
+            {
+                testString = Console.ReadLine();
+
+                if (!IsAllLetters(testString))
+                {
+                    Console.WriteLine("You entereted that wrong. Please try again.\n");
+                }
+                
+                else
+                {
+                    stop = false;
+                }
+            }
+            
+            return testString;
+        }
+
+        //check each character in string to see if it has any numbers
+        public static bool IsAllLetters(string testForLetters)
+        {
+            foreach (char c in testForLetters)
+            {
+               
+                if (!Char.IsLetter(c))
+                {
+                    return false;
+                }
+            }
+         
+            return true;
+        }
+
+        //Make sure string has only numbers
+        public static string CheckStringForDigitsOnly()
+        {
+            bool stop = true;
+            string testDigits = "";
+
+            while (stop)
+            {
+                testDigits = Console.ReadLine();
+
+                if (!IsAllDigits(testDigits))
+                {
+                    Console.WriteLine("You entereted that wrong. Please try again.\n");
+                }
+                else
+                {
+                    stop = false;
+                }
+            }
+
+            return testDigits;
+        }
+
+        //Make sure string is only numbers also allowed to have '.'
+        public static bool IsAllDigits(string testForNumbers)
+        {
+            foreach (char c in testForNumbers)
+            {
+                if (!Char.IsDigit(c) && c != '.')
+                {
+                    return false;
+                }
+            }
+            return true;
+        }
+
+
+        public static float CalculateBmi(float weightInKg, float height)
+        {
+            float bmi = weightInKg / (height * height);
+
+            Console.WriteLine($"\nYour BMI based on your heigh of {height}m and your weight of {weightInKg}Kg " +
+                             $"is {bmi} kg/(m*m)");
+            return bmi;
+        }
+
+
+
+
+
+        //I know there will be 2 inputs so I count to get right return string
+        static int count;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove the unused private field 'count'.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4852921)